pages/platform/databases/opensearch_03_advanced_parameters_references/guide.en-gb.md

```diff
@@ -0,0 +1,81 @@
+
+Please refer to this [guide](https://docs.ovh.com/gb/en/publiccloud/databases/advanced-configuration/#using-api) to find out how to change your advanced parameters from the OVHcloud API.
+
+
+### Advanced parameters for OpenSearch
+
+Below you can find a summary of every configuration option available for OpenSearch service:
+
+| Parameter | Value Type | Description |
+|:---|:---:|:---|
+| `opensearch.action_auto_create_index_enabled` | boolean | Explicitly allow or block automatic creation of indices. Defaults to true |
+| `opensearch.cluster_max_shards_per_node` | long | Controls the number of shards allowed in the cluster per data node |
+| `opensearch.cluster_routing_allocation_node_concurrent_recoveries` | long | How many concurrent incoming/outgoing shard recoveries (normally replicas) are allowed to happen on a node. Defaults to 2. |
+| `opensearch.http_max_content_length` | long | Maximum content length for HTTP requests to the OpenSearch HTTP API, in bytes. |
+| `opensearch.http_max_header_size` | long | The max size of allowed headers, in bytes |
+| `opensearch.http_max_initial_line_length` | long | The max length of an HTTP URL, in bytes |
+| `opensearch.indices_memory_index_buffer_size` | long | Percentage value. Default is 10%. Total amount of heap used for indexing buffer, before writing segments to disk. This is an expert setting. Too low value will slow down indexing; too high value will increase indexing performance but causes performance issues for query performance. |
+| `opensearch.indices_queries_cache_size` | long | Percentage value. Default is 10%. Maximum amount of heap used for query cache. This is an expert setting. Too low value will decrease query performance and increase performance for other operations; too high value will cause issues with other OpenSearch functionality. |
+| `opensearch.indices_query_bool_max_clause_count` | long | Maximum number of clauses Lucene BooleanQuery can have. The default value (1024) is relatively high, and increasing it may cause performance issues. Investigate other approaches first before increasing this value. |
+| `opensearch.indices_recovery_max_bytes_per_sec` | long | Limits total inbound and outbound recovery traffic for each node. Applies to both peer recoveries as well as snapshot recoveries (i.e., restores from a snapshot). Defaults to 40mb |
+| `opensearch.indices_recovery_max_concurrent_file_chunks` | long | Number of file chunks sent in parallel for each recovery. Defaults to 2. |
+| `opensearch.override_main_response_version` | boolean | Compatibility mode sets OpenSearch to report its version as 7.10 so clients continue to work. Default is false |
+| `opensearch.script_max_compilations_rate` | string | Script compilation circuit breaker limits the number of inline script compilations within a period of time. Default is use-context |
+| `opensearch.thread_pool_analyze_queue_size` | long | Size for the thread pool queue. See documentation for exact details. |
```

Comment:
Should a link be inserted here and for all "See documentation" occurrences below ?